src/util/search.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * SelectMultipleFilter needs to parse values from format
+ * "has_all:a,b,c,d" or "a,b,c,d"
+ */
+export const parseSelectFilterOptions = uriComponentValue => {
+  return uriComponentValue && uriComponentValue.indexOf(':') >= 0
+    ? uriComponentValue.split(':')[1].split(',')
+    : uriComponentValue && uriComponentValue.indexOf(':') < 0
+    ? uriComponentValue.split(',')
+    : [];
+};
+
+const flatten = (acc, val) => acc.concat(val);
+
+/**
+ * Check if the filter is currently active.
+ */
+export const findActiveFilter = (filterIds, urlQueryParams, filterConfigs) => {
+  const activeQueryParamKeys = filterConfigs
+    .filter(config => filterIds.includes(config.id))
+    .map(config => config.queryParamNames)
+    .reduce(flatten, []);
+
+  const paramEntries = Object.entries(urlQueryParams);
+  const activeKey = paramEntries.find(entry => {
+    const [key, value] = entry;
+    return activeQueryParamKeys.includes(key) && value != null;
+  });
+  return activeKey;
+};
```

Comment:
I'm quite confused of this function. What does it do? The comment says:

> Check if the filter is currently active.

but you don't pass it a single filter to check if it's active.

Then, it only returns the first active filter it finds. Is that supposed to work like that?

Reply:
Hmm. Yeah, this is confusing! A bunch of code that wasn't refactored/cleand after it ended up to this 'util' file.

The original idea was to check if any of the given filters (defined by filterIds) are active. I.e. if urlQueryParams include queryParamNames defined in filterConfig. 
I'll change the function name and its variable names a bit - and I'll add a better comment.